apps/web/src/app/(private)/datasets/_components/DatasetsTable/index.tsx

```diff
@@ -94,9 +94,7 @@ export function DatasetsTable({
                       />
                     }
                   >
-                    <Text.H6B color='white'>
-                      Show file preview (first 100 rows)
-                    </Text.H6B>
```

Comment:
Don't use `<Text>` components for simple content. Tooltip takes care of handling it based on variant and dark mode